components/apimgt/api-store-web/src/site/themes/fancy/templates/api/throttling/template.jag

```diff
@@ -120,10 +120,11 @@ var i, length, app, thumbURL, api = outputs.api, templatesList = outputs.uriTemp
 
 
                                 <% getAuthVal=putAuthVal=postAuthVal=deleteAuthVal=optionAuthVal=""; %>
+                                <% getThrottlingTier=postThrottlingTier=deleteThrottlingTier=optionsThrottlingTier=putThrottlingTier=""; %>
```

Comment:
if the intention is to clear the values of the variables, better assigning undefined to the variable rather than an empty string. For better readability, format the statement with spaces as follows.
getAuthVal = putAuthVal = postAuthVal = deleteAuthVal = optionAuthVal = undefined;